cities/schemas.py

```diff
@@ -0,0 +1,17 @@
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str | None = None
```

Comment:
The type hint `str | None` requires Python 3.10 or later. If you're using an earlier version, consider using `Optional[str]` from the `typing` module.